### PR TITLE
Badges & buttons

### PR DESCRIPTION
**Pull request by sanathkumar26700**
Created documentaion for,
1. Badges : 
       a. for avatar
       b. for headings
       c. for icons

2. Buttons : 
       a. Various Types
       b. various types with animations
       c. Special buttons

### CODE REVIEW

File: Components/Badges components/badges.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    <title>View.I</title>
+</head>
+
+<body>
+    <div class="header__container box-shadow">
+        <div class=" logo ">
+            <a class="anchor-style--none " href="/index.html ">
+                <span class="logo--title logo--title-black ">View<span class="dot ">.</span>I
+                <img src="/images/logo--eye.png " alt=" " class="logo--img img-responsive "> </span>
```
Comment (mimansha-swarup) [inline Components/Badges components/badges.html]: don't keep alt empty

File: Components/CSS/variables.css [hunk 2]
```diff
@@ -3,6 +3,13 @@
     --primary-color-dark: #7c3a00;
     --primary-color-hover: #b3621c;
     --secondary-color: #000;
+}
+
+:root {
```
Comment (mimansha-swarup) [inline Components/CSS/variables.css:8]: why are you maintaining two-variable files?

File: CSS/utilities/utilities.css [hunk 3]
```diff
@@ -28,14 +28,40 @@
...
+    background-color: var(--container-color);
+    padding: 1.6rem;
+    margin-bottom: 1.5rem;
+}
+
+.container-flex--mini {
+    display: flex;
+    gap: 0.5rem;
+    flex-flow: row wrap;
+    justify-content: center;
```
Comment (mimansha-swarup) [inline CSS/utilities/utilities.css]: you are using flex center too many time you can also make util class rather than repeating

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

File: CSS/pages/landing-page.css [hunk 4]
```diff
@@ -45,7 +45,7 @@ body {
     width: 50%;
 }
 
-.btn__outlined {
+.btn__outlined-white {
     color: white !important;
```
Comment (srishti-maurya) [inline CSS/pages/landing-page.css:49]: you may avoid using `!important` in your code

**Review by srishti-maurya — APPROVED** [review]
looks good

File: Components/CSS/variables.css (hunk 2 above)
Comment (sanathkumar26700) [reply Components/CSS/variables.css:8]: One for component library which will be imported by the user of the components, another variable file is just for the documentation and landing page which are not used for the components.

File: Components/Badges components/badges.html (hunk 1 above)
Comment (sanathkumar26700) [reply Components/Badges components/badges.html]: I must have missed it, thanks for pointing it out!

File: CSS/utilities/utilities.css (hunk 3 above)
Comment (sanathkumar26700) [reply CSS/utilities/utilities.css]: Thanks for the suggestion I'll look into it!

**Comment by sanathkumar26700** [comment]
> looks good

Thanks for the review!

**Comment by KaustubhM1997** [comment]
Looks good to me!

**Comment by sanathkumar26700** [comment]
> Looks good to me!

Thanks for the review!

**Pull request merged**